app/helpers/releases_helper.rb

```diff
@@ -39,4 +39,12 @@ def link_to_deploy_stage(stage, release)
       link_to stage.name, project_stage_deploys_path(@project, stage, deploy: deploy_params), method: :post
     end
   end
+
+  def github_user_avatar(github_user)
```

Comment:
reuse in `app/views/changeset/_commits.html.erb:9``app/views/changelogs/show.html.erb:33` so we get consistent images ... maybe move the 20x20 into css too

Reply:
:+1:

It's nice to have the size in the HTML, since it causes less jumping around. It also lets us parameterize it so that it matches the avatar size, since that's not the same in all cases.